roles/relay/tasks/main.yml

```diff
@@ -39,29 +39,11 @@
     dest: "{{ homedir }}/services/relay/nginx-redirect.conf"
     mode: '0644'
 
-
-- name: Copy strfry.conf to relay dir
```

Comment:
Are we just bundling strfry.conf in our docker build? Or is it being copied some other way?